requirements-ci.txt

```
@@ -1,5 +1,5 @@
 pennylane>=0.38
-qiskit
+qiskit<=1.2.4
```

In case qiskit 1.2 has a bug fix release.
```suggestion
qiskit<1.3
```